src/components/video-embed/index.tsx

```diff
@@ -0,0 +1,43 @@
+import ReactPlayer from 'react-player'
+import { VideoEmbedProps } from './types'
+import s from './video-embed.module.css'
+
+export default function VideoEmbed({
+  start,
```

Comment:
Ported from `learn`, but also converted to TypeScript and tried to get more specific with what's happening with props.